package.json

```diff
@@ -7,6 +7,7 @@
   "dependencies": {
     "@absinthe/socket": "^0.2.1",
     "@absinthe/socket-apollo-link": "^0.2.1",
+    "@apollo/client": "^3.6.8",
```

Comment:
Does this replace the old `apollo-client` library? If so, can you remove the old libraries?

Reply:
One of the tests in jest apparently relies on `@apollo/client` specifically, and we are using the `apollo-client` library. I played around with it and I couldn't find a way to make the test rely on the `apollo-client` library; it seems like there was some sub-dependency there that was specifically trying to call `@apollo/client` as modules were getting updated